### ISSUE TITLE
Also translate dish type into English

### ISSUE THREAD

**Issue by Friendly-Banana** [TUM-Dev/eat-api#253]

**Comment by COM8** [comment]
@Friendly-Banana sounds like a really good idea. Any chance you would like to work on a PR for it?

**Comment by Friendly-Banana** [comment]
Yes, but I'm not so familiar with the codebase. How would I go about adding this?
There seem to be multiple partial DishType collections: `FMIBistroMenuParser.DishType` https://github.com/TUM-Dev/eat-api/blob/01a2703104736872be191ccc3a04df72e94df3ad/src/menu_parser.py#L312 https://github.com/TUM-Dev/eat-api/blob/01a2703104736872be191ccc3a04df72e94df3ad/src/menu_parser.py#L743 https://github.com/TUM-Dev/eat-api/blob/01a2703104736872be191ccc3a04df72e94df3ad/src/menu_parser.py#L899

**Comment by COM8** [comment]
Basically yes. There is not really more to it.
